### DISCUSSION TITLE
How to use published functionality?

### DISCUSSION THREAD

**Discussion by AlexanderFalkenberg** [statamic-rad-pack/runway#294] [Feature Requests]
Traditional entries have a published toggle, which is missing using runway models leaving a shadow border. Is it possible to use the publish functionality or remove the border?

**Comment by duncanmcclean** [comment]
Runway doesn't have concept "published" models, so this border should be removed. I've created an issue for this: #295.

If you need published models in your project, you can add a toggle field to models yourself, then handle the filtering on the front-end when using the Antlers Tag.

**Reply by AlexanderFalkenberg** [reply]
Thanks!

**Comment by duncanmcclean** [comment]
Hey! 👋

Just following up on this discussion to say this is now possible. Runway now has native support for publishing/unpublishing entries, and all of the relevant filtering for that in Antlers/GraphQL/REST API.

See https://github.com/statamic-rad-pack/runway/pull/506.
